modules/migration-error-messages.adoc

```diff
@@ -12,37 +12,42 @@ This section describes common error messages you might encounter with the {mtc-f
 [id="ca-certificate-error-in-console_{context}"]
 == CA certificate error in the {mtc-short} console
 
-If a `CA certificate error` message is displayed the first time you try to access the {mtc-short} console, the likely cause is the use of self-signed CA certificates in one of the clusters.
+If the {mtc-short} console displays a `CA certificate error` message the first time you try to access it, the likely cause is that a cluster uses self-signed CA certificates.
 
-To resolve this issue, navigate to the `oauth-authorization-server` URL displayed in the error message and accept the certificate. To resolve this issue permanently, add the certificate to the trust store of your web browser.
+To resolve this issue, navigate to the `oauth-authorization-server` URL in the error message and accept the certificate.
 
-If an `Unauthorized` message is displayed after you have accepted the certificate, navigate to the {mtc-short} console and refresh the web page.
+To resolve this issue permanently, install the certificate authority so that it is trusted.
+
+If the browser displays an `Unauthorized` message after you have accepted the CA certificate, navigate to the {mtc-short} console and then refresh the web page.
 
 [id="oauth-timeout-error-in-console_{context}"]
 == OAuth timeout error in the {mtc-short} console
 
-If a `connection has timed out` message is displayed in the {mtc-short} console after you have accepted a self-signed certificate, the causes are likely to be the following:
+If the {mtc-short} console displays a `connection has timed out` message after you have accepted a self-signed certificate, the causes are likely to be the following:
 
 * Interrupted network access to the OAuth server
 * Interrupted network access to the {product-title} console
 * Proxy configuration that blocks access to the `oauth-authorization-server` URL. See link:https://access.redhat.com/solutions/5514491[MTC console inaccessible because of OAuth timeout error] for details.
 
-You can determine the cause of the timeout.
+To troubleshoot this error, check the following:
 
-.Procedure
+* Inspect the {mtc-short} console web page with a browser web inspector.
+* Check the `Migration UI` pod log for errors.
 
-. Navigate to the {mtc-short} console and inspect the elements with the browser web inspector.
-. Check the `MigrationUI` pod log:
-+
-[source,terminal]
-----
-$ oc logs <MigrationUI_Pod> -n openshift-migration
-----
+[id="backup-storage-location-errors-in-velero-pod-log_{context}"]
+== Backup storage location errors in the Velero pod log
+
+If a Velero backup references a backup storage location (BSL) that does not exist, the `Velero` pod log might display the following error messages:
+
+* `Error checking repository for stale locks`
+* `Error getting backup storage location: backupstoragelocation.velero.io \"my-bsl\" not found`
```

Comment:
The last two letters of this error message are getting cut off (the "n" and "d" of "found").

Reply:
I just noticed that myself. Very odd. I could force a line break.